src/ThgHostingTest.php

```diff
@@ -0,0 +1,118 @@
+<?php declare(strict_types=1);
+/**
+ * (c) The Hut Group 2001-2019, All Rights Reserved.
+ *
+ * This source code is the property of The Hut Group, registered address:
+ *
+ * 5th Floor, Voyager House, Chicago Avenue, Manchester Airport,
+ * Manchester, England, M90 3DQ
+ * @author Michal Dzierzbicki <michal.dzierzbicki@thg.com>
+ */
+namespace ThgHosting;
+
+class ThgHostingTest extends ThgHostingClient
+{
+    public const PROD_ENV  = 'prod';
+    public const DEV_ENV   = 'dev';
+    public const LOCAL_ENV = 'local';
+
+    public function __construct(string $env = self::DEV_ENV)
```

Comment:
I don't know why you didn't make env variable called Enviorment which would decide what enviroment it is. local/stable/prod. This works but is just odd